roles/openshift_master/templates/native-cluster/atomic-openshift-master-controllers.service.j2

```diff
@@ -11,6 +11,21 @@ Type=notify
 EnvironmentFile=/etc/sysconfig/{{ openshift_service_type }}-master-controllers
 Environment=GOTRACEBACK=crash
 ExecStart=/usr/bin/openshift start master controllers --config=${CONFIG_FILE} $OPTIONS
+ExecStart=/usr/bin/hyperkube kube-controller-manager \
+  --controllers="*" --controllers=-ttl --controllers=-bootstrapsigner --controllers=-tokencleaner --controllers=-horizontalpodautoscaling --controllers=-serviceaccount-token \
+  --service-account-private-key-file=openshift.local.config/master/serviceaccounts.private.key \
```

Comment:
Do things like `openshift.local.config` not need to be `{{ openshift.local.config }}` ?  Throughout?

Reply:
> Do things like openshift.local.config not need to be {{ openshift.local.config }} ? Throughout?

I don't know.  @sdodson ?

Reply:
It should be `--service-account-private-key-file={{ openshift_master_config_dir }}/serviceaccounts.private.key` which would evaluate to /etc/origin/master/serviceaccounts.private.key